scarlet/source.py

```diff
@@ -227,6 +227,36 @@ def init_multicomponent_source(sky_coord, frame, observation, bg_rms, flux_perce
     return seds, morphs
 
 
+class RandomSource(Component):
+    """Sources with uniform random morphology.
+
+    For cases with no well-defined spatial shape, this source initializes
+    a uniform random field and (optionally) matches the SED to match a given
+    observation.
+    """
+    def __init__(self, frame, observation=None, **component_kwargs):
+        """Source intialized with a single pixel
+
+        Parameters
+        ----------
+        frame: `~scarlet.Frame`
+            The frame of the model
+        observation: list of `~scarlet.Observation`
+            Observation to initialize the SED of the source
+        component_kwargs: dict
+            Keyword arguments to pass to the component initialization.
+        """
+        C, Ny, Nx = frame.shape
+        morph = np.random.rand(Ny, Nx)
+
+        if observation is None:
+            sed = np.random.rand(C)
```

Comment:
I'm not sure we want to have both seds and morphologies initialised at random. Also, not providing an observation would not make much sense, would it?

Reply:
True, this is why you *can* provide an observation, but you don't have to. Double random initialization is not uncommon for matrix factorization, but normally one would scale to the amplitude in each channel, which is what this code will do if given an observation.

In some sense, this part demonstrates that a source can formally exist without an observation to initialize it.

Reply:
I just don't see a use case for no observation initialisation. 

If no observation is given, then nothing is scaled (to nothing because there is nothing to scale to, makes sense...). That's about the worse case scenario for random initialisation. This is actually great for studying random initialisation, but not for production, we agree?